src/pretix/static/pretixcontrol/js/ui/main.js

```diff
@@ -6,6 +6,44 @@ function question_page_toggle_view() {
 
     show = $("#id_type").val() == "B" && $("#id_required").prop("checked");
     $(".alert-required-boolean").toggle(show);
+
+    let input = $('#id_default_value');
```

Comment:
Could you wrap this in a function of if statement that only executes on the question edit page? Mostly to make it obvious to future developers what this is used for.

Reply:
As far as I can see, the `question_page_toggle_view` function is already only executed on the question edit page, right? Of course, I can still extract this piece in a function that is then executed from `question_page_toggle_view`.